<launch>

    <node name="adaptive_super_twisting" pkg="sensors" type="adaptive_super_twisting" >
    <param name = "k_u" value = "0.2" />
	<param name = "k_psi" value = "0.4" />
	<param name = "kmin_u" value = "0.01" />
	<param name = "kmin_psi" value = "0.01" />
	<param name = "epsilon_u" value = "0.05" />
    <param name = "epsilon_psi" value = "0.05" />
	<param name = "mu_u" value = "0.05" />
	<param name = "mu_psi" value = "0.05" />
	<param name = "lambda_u" value = "0.001" />
	<param name = "lambda_psi" value = "1" />
    </node>

</launch>
